### PR TITLE
chore: adds other button examples to explorer

### PR DESCRIPTION
**Pull request by nalogiudice**
Closes [#2084 ](https://github.com/microsoft/fast-dna/issues/2084)
# Description
Adds button examples to explorer.

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

**Comment by nalogiudice** [comment]
Can someone please smoke? I am having issues trying to run the component explorer server so I can't test, but the data _seems_ correct.

File: packages/fast-component-explorer/app/utilities/configs/lightweight-button.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        },
+        {
+            displayName: "Disabled",
+            data: {
+                disabled: true,
+                children: "Disabled lightweight button",
+            },
+        },
+        {
+            displayName: "Before content lightweight button",
```
Comment (janechu) [inline packages/fast-component-explorer/app/utilities/configs/lightweight-button.ts]: This doesn't read well, maybe "Glyph before text"

File: packages/fast-component-explorer/app/utilities/configs/neutral-button.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        },
+        {
+            displayName: "Disabled",
+            data: {
+                disabled: true,
+                children: "Disabled neutral button",
+            },
+        },
+        {
+            displayName: "Before content neutral button",
```
Comment (janechu) [inline packages/fast-component-explorer/app/utilities/configs/neutral-button.ts]: same issue here, would rather have "Glyph before text" and below "Glyph after text"

File: packages/fast-component-explorer/app/utilities/configs/lightweight-button.ts (hunk 1 above)
Comment (nalogiudice) [reply packages/fast-component-explorer/app/utilities/configs/lightweight-button.ts]: I agree it reads strange, but `beforeContent` and `afterContent` could potentially be something other than a glyph. How about "Content before text"?

Comment (nalogiudice) [reply packages/fast-component-explorer/app/utilities/configs/lightweight-button.ts]: Discussed offline and updated.

**Pull request merged**